_rules/printable-characters-shortcut-ffbc54.md

```diff
@@ -174,7 +177,7 @@ This [HTML document][] has a [keyboard shortcut][] using only a [printable chara
 
 #### Passed Example 5
```

Comment:
Passed Examples 5-6 feel a bit redundant (they illustrate the same think as Passed Example 2-3).

Reply:
They illustrate the same, but for different parts of the expectation. Examples 2-3 are for the **disable** mechanism, and examples 5-6 are for the **remap** mechanism.